src/core/99_variables.tf

```
@@ -790,3 +795,46 @@ variable "function_cgn_autoscale_default" {
   description = "The number of instances that are available for scaling if metrics are not available for evaluation."
   default     = 1
 }
+
+# Shared functions
+variable "plan_shared_kind" {
+  type        = string
+  description = "App service plan kind"
+  default     = null
+}
+
+variable "plan_shared_sku_tier" {
```

```suggestion
variable "plan_shared_1_sku_tier" {
```
add 1 to respect naming convention

---

hi @pasqualedevita! Can you direct us to the documentation that contains the proper naming scheme? We are trying to be as independent as we can (this knowlede could prevent you some saturday work ;-)

---

solved in https://github.com/pagopa/io-infra/pull/316/commits/46bd3249e99fe4ac8b3562c8c766c5af51e804da